Src/CMakeLists.txt

```diff
@@ -9,6 +9,49 @@ else()
 endif()
 
 add_subdirectory(JxrDecode)
+
+# if the build is configured to include the curl-based stream, we need to find or download the curl library
+if (LIBCZI_BUILD_CURL_BASED_STREAM)
+  if (LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_LIBCURL)
+    find_package(CURL CONFIG QUIET)
+    if (NOT CURL_FOUND)
+      message(FATAL_ERROR [=[
+        CURL library was not found, which is required for building. Consider installing it with a package manager, something
+        like 'sudo apt-get install libcurl4-openssl-dev', or disable the option 'LIBCZI_BUILD_PREFER_EXTERNALPACKAGE_LIBCURL'.
```

Comment:
We may also want to mention vcpkg here

Reply:
I added a small note which mentions vcpkg in the build-instructions markdown.